README.md

```diff
@@ -175,39 +216,47 @@ The generated Event Management Agent jar file is found in:
 application/target/event-management-agent-1.0.0-SNAPSHOT.jar
 ```
 
-## Deployment
+### Running the Event Management Agent as a process
 
-There are two main modes of deployment:
+You can run the Event Management Agent as a process in both connected and standalone modes.
 
-* Connected: The Event Management Agent connects to the Solace PubSub+ Event Portal. Scans are initiated from Event Portal. Completed scan results are uploaded to Event Portal automatically.
+Specify:
+* the Event Management Agent jar file (e.g. event-management-agent-1.0.0-SNAPSHOT.jar)
+* the location of the connection file (e.g. /path/to/file/AcmeRetail.yml) - See [Generating the Event Management Agent connection file](#generating-the-event-management-agent-connection-file) to generate a connection file
 
-* Standalone: The Event Management Agent is controlled via the REST API and results must be uploaded manually.
+```
+java -jar application/target/event-management-agent-1.0.0-SNAPSHOT.jar --spring.config.location=/path/to/file/AcmeRetail.yml
+```
 
-## Running a scan
+The Event Management Agent is ready.
 
-### REST interface
+# Broker Plugins
 
-The Event Management Agent includes a REST API that allows the user to initiate a scan. Each plugin requires its own
-custom set of authentication and identification attributes that must be supplied by the user.
+The Event Management Agent comes with the following event or message broker plugins included:
 
-See [REST Documentation](docs/rest.md) for additional information
+* Apache Kafka
```

Comment:
Do we want to add Confluent Kafka here since we called it out above?

Reply:
Done